server/util/src/lib.rs

```diff
@@ -27,3 +29,9 @@ pub use error::*;
 
 mod file;
 pub use file::*;
+
+pub fn is_central_server() -> bool {
```

Comment:
It was a bit of a mission passing it around, and I wasn't yet sure how this would be specified (via feature flag, env variable, a settings that syncs etc..). Also was thinking about global multi read one writer mutex for this sort of thing, added to [carry over](https://github.com/msupply-foundation/open-msupply/issues/2821) as comment

Reply:
Was mainly thinking about alterntive setups, e.g when creating a test setup where you might not want to use an env variable. For things like that it might be useful have configurable.

Having a global variable for it is probably ok but still have the feeling its "cleaner" to pass it arround, e.g. the immutable settings object we already have seems suitable and is already available through the SettingService.